xbmc/guilib/GUIWindowManager.cpp

```diff
@@ -1013,7 +1017,40 @@ void CGUIWindowManager::OnApplicationMessage(ThreadMessage* pMsg)
     }
     pMsg->SetResult(static_cast<int>(dialogOK->IsConfirmed()));
   }
+
+  case TMSG_GUI_SETPROPERTY:
+  {
+
+    if (pMsg->lpVoid)
+    {
+      const CVariant* params = static_cast<const CVariant*>(pMsg->lpVoid);
+      const std::string& key = (*params)["key"].asString();
+      const CVariant& value = (*params)["value"];
+      int iWindowID = CWindowTranslator::TranslateWindow((*params)["window"].asString());
+
+      if (iWindowID == -1)
+      {
+        iWindowID = GetActiveWindowOrDialog();
+      }
+
+      CGUIWindow* pNewWindow = GetWindow(iWindowID);
+      if (pNewWindow)
+      {
+        pNewWindow->SetProperty(key, value);
+        pNewWindow->SetInvalid();
+      }
+      else
+      { // nothing to see here - move along
+        CLog::Log(LOGERROR, "Unable to locate window with id %d.  Check skin files",
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)